bitnami/airflow/templates/deployment-scheduler.yaml

```diff
@@ -58,6 +58,8 @@ spec:
           volumeMounts:
             - name: git-cloned-dag-files
               mountPath: /dags
+            - name: dag-git-keys
+              mountPath: /root/.ssh
```

Comment:
This volume should only be mounted if `airflow.cloneDagFilesFromGit.sshKeySecret` is set.